oam/install_scripts/columnstore-pre-uninstall.in

```diff
@@ -22,29 +21,32 @@ else
     kill $(pidof $PROGS) > /dev/null
     sleep 3
     kill -9 $(pidof $PROGS) > /dev/null
+    clearShm  > /dev/null 2>&1
 fi
 
-rm -f /etc/profile.d/columnstoreAlias.sh
+if [ -n "$(pgrep -x ProcMon)" ] || [ -n "$(pgrep -x ProcMgr)" ];then
+   # Old system must be running, kill ProcMon/ProcMgr 
+    pkill ProcMon
+    pkill ProcMgr
+fi
```

Comment:
Let us make sure that an old systemd gets rid of running ProcMon/ProcMgr.
columnstore-pre-uninstall is also called during installation as a "pre-install" script.